mpqp/core/instruction/measurement/expectation_value.py

```diff
@@ -194,8 +194,8 @@ class ExpectationMeasure(Measure):
 
     def __init__(
         self,
-        targets: list[int],
         observable: Observable,
+        targets: Optional[list[int]] = None,
         shots: int = 0,
```

Comment:
Shouldn't we put `shots: int = 1024` like we did for `BasisMeasure` ?

Reply:
We had a discussion about that: in basis measure, it is less useful to specify shots=0. Indeed, this would mean you are basically asking for a state vector job, which is equivalent to putting no measure. So in the case someone wants a state vector job, we would suggest putting no measure, rather that a basis measure with shots=0. On the other hand, for the expectation measure, this is not so true, so having the default being 0 is not crazy to me

Reply:
on the other hand, having different defaults for different measures might be weird, indeed

Reply:
For all measurements, say in the doc that shots=0 means no shot noise

Reply:
I agree with this, we can let it as it is.